02/studenttests.ml

```diff
@@ -361,6 +361,36 @@ let instruction_tests = [
     (fun m -> m.regs.(rind Rax) = -20L));
 ]
 
+
+(*GEORGR'S SIMPLE INSTRUCTION TESTS*)
```

Comment:
I think you can remove this commenti since you already put them in your own group